.env.test

```diff
@@ -11,6 +11,8 @@ export GITHUB_ORG_NAME="isomerpages"
 export GITHUB_BUILD_ORG_NAME="opengovsg"
 export GITHUB_BUILD_REPO_NAME="isomer-build"
 export ISOMERPAGES_REPO_PAGE_COUNT=3
+export MAX_NUM_OTP_ATTEMPTS=5
```

Comment:
ooof I thought we removed tracking of `.env.test` HAHA

Reply:
apparently needed this for ci-tests to run

Reply:
as in `.env.test` git ignored [here](bd95543b749d185f66d9e9a0c9a5cee839df716d) haha -> could we add this file in 1pw?

Reply:
I mean, the CI command to run is `source .env.test && jest` right. Then how will the ci-test run if we remove the .env.test?

Reply:
supposed to have `.env.test` cos it's used for testing and we **shouldn't** be using secrets for testing anyway